gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #142 Squads Front End

### Description

I created the Squad model. I also did the front end and back end for `/s/<sid>` (`squad.html`) and `/s/<sid>/edit` (`edit_squad.html`).

### Testing Directions

```
./setup
python3 manage.py runserver
```

1.  Log in as a user. Click on "My Squad" in the navbar.
1.  A squad containing only yourself should be present. Click on "Edit Your Squad".
1.  Add emails. Valid emails will redirect you back to `/s/<sid>`. Invalid emails like emails that don't exist or emails that are already in your squad will just refresh the page. I put `TODO` comments in `showup/views.py` to show where messages need to be written.
1.  Make 2 squads of size 2. See that when you add an email of one of the squads, all the users are combined into a squad of 4.
1.  Verify the DB using `/admin`. I wrote the code so that the emails you add are joined to your squad, and the old squad is deleted.

### Miscellaneous Comments

*   #170 improves the UI.

### Checklist
